web/carousel/src/components/PasswordForm/PasswordForm.js

```diff
@@ -73,7 +83,7 @@ const PasswordForm = (props) => {
             style={passLength ? { color: "forestgreen" } : { color: "red" }}
             className={classes.PasswordCheck}
           >
-            Password should be between 6-20 characters.
+            Password should be between 6-20 letters.
```

Comment:
I think should stay at characters, as we count digits as well and they are not letters

Reply:
Yes, you are right. I forgot to change it back.